wagn/lib/wagn/tasks/test.rake

```diff
@@ -74,7 +74,7 @@ namespace :test do
     ActiveRecord::Base.establish_connection
     tables.each do |table_name|
       i = "000"
-      File.open("#{Wagn.gem_root}/test/fixtures/#{table_name}.yml", 'w') do |file|
+      File.open("#{Cardio.gem_root}/test/fixtures/#{table_name}.yml", 'w') do |file|
         data = ActiveRecord::Base.connection.select_all(sql % table_name)
         file.write data.inject({}) { |hash, record|
           record['trash'] = false if record.has_key? 'trash'
```

Comment:
He referred to line 2 and 6 in the top comment.  That code reads, updates and writes database.yml, which is pretty special purpose.  At first I thought it was related to generating an app, but it is for configuring the database names in database.yml.  Is that for an app generated by a test script?  That would make some sense.  Otherwise, why are we generating a database.yml in a rake task?